### DISCUSSION TITLE
smartbox turbo+ with EFR32MG1B232GG on board, flash the firmware 'router-zigbee'

### DISCUSSION THREAD

**Discussion by kva4991** [grobasoz/zigbee-firmware#39] [Q&A] [answered]
Hello, please help me deal with Zigbee EFR32MG12 inside the router "smartbox turbo+".
In particular, the article describes the firmware "router-zigbee" ( https://github.com/grobasoz/zigbee-firmware/blob/master/MG1B232/NCP_USW_MG1B232_678_115k2_TX-PA0_RX-PA1_BT-PD15.gbl ) this is what I need.
I transmitted it via xmodem, but I can't detect this zigbee router with my zigbee coordinator (both devices are identical in hardware).
I guess I need to press the pairing button, but I couldn't figure out where it is on the "smartbox turbo+"
Please help me find this function/button.

P.S.: I used the article for the firmware: https://mysku.club/blog/diy/84501.html
![изображение](https://user-images.githubusercontent.com/31066698/208183434-314f7c14-2347-4119-ac80-ede68e3ce63d.png)

**Comment by Hedda** [comment, marked as answer]
You are probably more likely to get a reply if instead post to https://github.com/zigpy/zigpy/discussions and/or https://community.home-assistant.io/
